OpenRA.Mods.Common/Traits/BotModules/HarvesterBotModule.cs

```diff
@@ -102,7 +106,6 @@ void IBotTick.BotTick(IBot bot)
 			scanForIdleHarvestersTicks = Info.ScanForIdleHarvestersInterval;
 
 			// Find new harvesters
-			// TODO: Look for a more performance-friendly way to update this list
 			var newHarvesters = world.ActorsHavingTrait<Harvester>().Where(a => !unitCannotBeOrdered(a) && !harvesters.ContainsKey(a));
```

Comment:
so why aren't we using the actor index to get these harvesters?

Reply:
this whole system is meant for tracking existing harvesters and can be replaced

Reply:
but we already have harvestersIndex in the class?

Reply:
That one also filters on `Info.HarvesterTypes` whereas this one doesn't. And given the desc on that of `Leave empty to disable harvester replacement. Currently only needed by harvester replacement system.` I don't want to apply it here, as that sounds like it could break any configs that were relying on being able to not specify the HarvesterTypes.